docs/hazmat/primitives/asymmetric/index.rst

```diff
@@ -32,4 +32,36 @@ and Elliptic Curve.
     serialization
     utils
 
+.. module:: cryptography.hazmat.primitives.asymmetric
+
```

Comment:
Mmm, I'm not sure the index is a good place for these, seems like it'll overshadow the content.